README.md

```diff
@@ -275,6 +275,7 @@ Additional lists you might find useful:
 *Search Engine Optimization.*
 
 - [Muffin/Slug plugin](https://github.com/UseMuffin/Slug) - A plugin for generating slugs and finding records by slug. Uses a pluggable architecture which allows using your own slug generator class.
+- [Seo plugin](https://github.com/orgasmicnightmare/cakephp-seo) - Manage seo tags with ease. It comes with a Behavior, a Component, a View Cell and the default admin crud actions for each tables.
```

Comment:
crud => CRUD imo.
But I wonder if you need to be that specific regarding Behavior, Component etc.
I think the description should not list the exact types of classes used, but the functionality this plugin does.